<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>etudiantWindow</class>
 <widget class="QMainWindow" name="etudiantWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#etudiantWindow {
background:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
#QSpinBox{

border-style: solid;
border:2px solid gray ;

border-radius: 8px;
}


#mainFrame {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
QLineEdit {
padding: 2px;
border-style: solid;
border:2px solid gray ;

border-radius: 8px;


}

QDateEdit 
{
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: white;

background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 13px;
padding-left: 5px;
padding-right: 5px;

}


QLabel {
color :  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
color : black;
font-weight: bold;
font-size: 12px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>480</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
background-color: transparent;
border-image: url(:/img/img/retour.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_espace">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>281</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_espace {
background-color: transparent;
border-image: url(:/img/img/espace etudiant.jpg);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3{
background-color: transparent;
border-image: url(:/img/img/chercher.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>            Information personnel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>382</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_4{
background-color: transparent;
border-image: url(:/img/img/resto.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>461</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QTableView" name="tableEtudiant">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>41</y>
        <width>89</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#550000;&quot;&gt;Pour payer frais :&lt;/span&gt;&lt;a href=&quot;http://esprit.tn/logement-et-restauration/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#aa5500;&quot;&gt;cliquez ici&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>41</y>
        <width>117</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>insérez votre id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>41</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QDateEdit" name="lineEdit_inserezmenu">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>31</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_note">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>136</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>insérez date menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_notezmenu">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>notez menu</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_menu2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>391</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_affichermenu">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>label_espace</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
